docs/src/main/asciidoc/quartz.adoc

```diff
@@ -135,15 +135,15 @@ import io.quarkus.scheduler.Scheduled;
 public class TaskBean {
 
     @Transactional
-    @Scheduled(every = "10s") <2>
+    @Scheduled(every = "10s", identity = "task-job") <2>
```

Comment:
@machi1990 I believe that the identity is required for clustering...

Reply:
It is better t have a user provided identity as can be controlled easily by them via migration tools. IIRC we auto-generate one if not provided. This change is safe from my point of view.

Reply:
Hm, I wasn't sure whether we generate the same identity for each cluster node though... but it seems we should for identical apps.